### DISCUSSION TITLE
Updating translation for next release

### DISCUSSION THREAD

**Discussion by mrtnbr** [bpatrik/pigallery2#722] [General]
Hi,

I'd like to update the Dutch translation before the next release is done. I saw you mention that there will be only bugfixes before the next release and no new features in the next release. 

When do you feel the time is right to do this? The reason I ask is that I'd like to do all the corrections in one go (as much as possible anyway).

Also, I feel that  [#594](https://github.com/bpatrik/pigallery2/issues/594) might be related to translation? I had a look but I can't figure out what to do to fix it myself yet.

Thanks!

Maarten

**Comment by bpatrik** [comment]
Lets wait with that a bit. I try update the translation files this week and open a bug for the repease candidate.

**Comment by bpatrik** [comment]
(hopefully) there wont be any more text changes before the next release.

See: https://github.com/bpatrik/pigallery2/issues/725
